migrations/v2.js

```diff
@@ -0,0 +1,225 @@
+  });
+
+  /**
+   * * Add field to each item in a JSON array and set blank.
+   */
+  mutateContent('adapt-contrib-assessment - add assessment._questions._canShowModelAnswer attribute', async () => {
+    assessments.forEach(assessment => {
+      assessment._questions.forEach(item => {
+        item._canShowModelAnswer = true;
+      });
+    });
+    return true;
+  });
+
+  checkContent('adapt-contrib-assessment - check assessment._questions._canShowModelAnswer attribute', async () => {
+    const isValid = assessments.every(assessment =>
+      assessment._questions.every(item =>
+        item._canShowModelAnswer !== false
+      )
+    );
+    if (!isValid) throw new Error('adapt-contrib-assessment - _canShowModelAnswer not added to every instance of assessment._questions');
+    return true;
+  });
+
+  updatePlugin('adapt-contrib-assessment - update to v2.0.3', { name: 'adapt-contrib-assessment', version: '2.0.3', framework: '>=2.0.0' });
+});
+
+describe('adapt-contrib-assessment - v2.0.3 > v2.1.0', async () => {
+  let articles, assessments;
+
+  whereFromPlugin('adapt-contrib-assessment - from v2.0.3', { name: 'adapt-contrib-assessment', version: '<2.1.0' });
+
+  whereContent('adapt-contrib-assessment - where assessment', async content => {
+    articles = content.filter(({ _type }) => _type === 'article');
+    assessments = articles.filter(({ _type, _assessment }) => _type === 'article' && _assessment !== undefined);
+    return assessments.length;
+  });
+
+  /**
+   * * Add JSON field and set attribute.
+   */
+  mutateContent('adapt-contrib-assessment - add assessment._suppressMarking', async () => {
+    assessments.forEach(assessment => {
+      assessment._suppressMarking = true;
+    });
+    return true;
+  });
+
+  checkContent('adapt-contrib-assessment - check assessment._suppressMarking attribute', async () => {
+    const isValid = assessments.every(assessment =>
+      assessment._suppressMarking === true
+    );
+    if (!isValid) throw new Error('adapt-contrib-assessment - _suppressMarking not added to every instance of assessment as true.');
+    return true;
+  });
+
+  updatePlugin('adapt-contrib-assessment - update to v2.1.0', { name: 'adapt-contrib-assessment', version: '2.1.0', framework: '>=2.0.18' });
+});
+
+describe('adapt-contrib-assessment - v2.1.0 > v2.1.1', async () => {
+  let articles, assessments;
+
+  whereFromPlugin('adapt-contrib-assessment - from v2.1.0', { name: 'adapt-contrib-assessment', version: '<2.1.1' });
+
+  whereContent('adapt-contrib-assessment - where assessment', async content => {
+    articles = content.filter(({ _type }) => _type === 'article');
+    assessments = articles.filter(({ _type, _assessment }) => _type === 'article' && _assessment !== undefined);
+    if (assessments) return true;
+  });
+
+  /**
+   * * Modify existing attribute to new value.
+   */
+  mutateContent('adapt-contrib-assessment - add assessment._isPercentageBased', async () => {
```

Comment:
I don't think the following properties were changed at all from `v2.1.0` to `v2.1.1`: `_isPercentageBased`, `_canShowFeedback`, `_canShowMarking`, and `_canShowModelAnswer`

Reply:
This is a weird one. Defaults look like like they were added to attributes in this.
https://github.com/adaptlearning/adapt-contrib-assessment/commit/14daa5ad1e9f2d654b8ed8fb97f18621d4290c55#diff-bbbc797064913d6074c5515fadbef42b3c90e0b677002d065500e190e0d3bf88R221

Reply:
@joe-replin I think the schema was reorganised, which makes it really difficult to read (hate when that happens!):, but you can see these attributes removed and added in the same diff

Reply:
Looks like the attributes were in but a default value added to a few of them, they were all added before v2.0.1 so I think we're safe to remove these as values will have been assigned to current courses. I'll pick this up while I'm adding the tests.